content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -24,7 +24,7 @@ To generate service-level metrics directly in the Envoy proxies, follow these st
 1.  To prevent duplicate telemetry generation, disable calls to `istio-telemetry` in the mesh:
 
     {{< text bash >}}
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --set mixer.telemetry.enabled=false --set mixer.policy.enabled=false
+    $ istioctl manifest --set values.mixer.telemetry.enabled=false --set values.mixer.policy.enabled=false
```

Comment:
Can be shortened to --set values.mixer.telemetry.enabled=false,values.mixer.policy.enabled=false.
Here and throughout.
Also istioctl manifest apply - here and throughout.